eventlet/green/http/__init__.py

```diff
@@ -52,8 +52,6 @@
 # 8. By copying, installing or otherwise using Python, Licensee
 # agrees to be bound by the terms and conditions of this License
 # Agreement.
-import six
-assert six.PY3, 'This is a Python 3 module'
```

Comment:
`assert` must not be used in code loaded at runtime, if user call python with the optimization flag, all asserts would be removed, and, so, this kind of code would be removed at runtime.

That's one less to transform.